src/components/Enrichment/Enrichment.jsx

```diff
@@ -2160,35 +2223,50 @@ class Enrichment extends Component {
                   />
                 </div>
                 <Grid.Column
-                  className="EnrichmentTableWrapper"
+                  className="EnrichmentTableWrapper two-col-sticky"
                   mobile={16}
                   tablet={16}
                   largeScreen={16}
                   widescreen={16}
                 >
-                  <EZGrid
-                    ref={this.EnrichmentGridRef}
-                    uniqueCacheKey={enrichmentCacheKey}
-                    data={enrichmentResults}
-                    columnsConfig={enrichmentColumns}
-                    // totalRows={rows}
-                    // use "rows" for itemsPerPage if you want all results. For dev, keep it lower so rendering is faster
-                    itemsPerPage={itemsPerPageEnrichmentTable}
-                    onItemsPerPageChange={this.handleItemsPerPageChange}
-                    loading={isEnrichmentTableLoading}
-                    // exportBaseName="Enrichment_Analysis"
-                    // columnReorder={this.props.columnReorder}
-                    disableColumnReorder
-                    disableGrouping
-                    disableColumnVisibilityToggle
-                    min-height="75vh"
-                    additionalTemplateInfo={
-                      additionalTemplateInfoEnrichmentTable
-                    }
-                    emptyMessage={CustomEmptyMessage}
-                    disableQuickViewEditing
-                    disableQuickViewMenu
-                  />
+                  <div
+                    ref={(el) => {
+                      if (el && !this.frozenColumnWrapperRef) {
+                        this.frozenColumnWrapperRef = el;
+                        this.setupFrozenColumnResizeObserver();
+                      } else if (!el && this.frozenColumnWrapperRef) {
+                        this.frozenColumnWrapperRef = null;
+                        if (this.frozenColumnResizeObserver) {
+                          this.frozenColumnResizeObserver.disconnect();
```

Comment:
Just make sure this runs on unmount - if not, you need to disconnect on unmount still since you removed that bit from onUnmount